metadata/planning/wilderness_proposal_red_rock.md

```diff
@@ -0,0 +1,73 @@
+# Title
+
+Utah Wilderness Proposal Red Rock
+
+## ID
+
+d3959e12-c08e-4104-a0f0-857f8cc33075
+
+## Brief Summary
+
+Polygon dataset of areas considered for wilderness designation under HR 3031.
```

Comment:
This dataset predates HR 3031. Googling it was first introduced in  the House in '89 and the Senate in '97. Wayne Owens probably had something to do with it.  The 8.4 million acres is an important number.